### ISSUE TITLE
Submitted workchains that contain an InlineCalculation raise an error.

### ISSUE THREAD

**Issue by greschd** [aiidateam/aiida_core#1250]
Example traceback (test added in https://github.com/greschd/aiida_core/tree/run_inlinecalc_in_submitted_workchain): 
```
03/08/2018 03:55:41 PM, aiida.orm.implementation.general.calculation.work.WorkCalculation: [REPORT] [1175|InlineCalcRunnerWorkChain|on_except]: Traceback (most recent call last):
  File "/home/greschd/.virtualenvs/aiida_dev/src/plumpy/plumpy/base_process.py", line 280, in _run
    result = self.run_fn(*self.args, **self.kwargs)
  File "/home/greschd/.virtualenvs/aiida_dev/src/plumpy/plumpy/processes.py", line 403, in run
    return self._run()
  File "/home/greschd/programming/aiida/aiida_core/aiida/work/workchain.py", line 120, in _run
    return self._do_step()
  File "/home/greschd/programming/aiida/aiida_core/aiida/work/workchain.py", line 131, in _do_step
    finished, return_value = self._stepper.step()
  File "/home/greschd/.virtualenvs/aiida_dev/src/plumpy/plumpy/workchains.py", line 224, in step
    return True, self._fn(self._workchain)
  File "/home/greschd/programming/aiida/aiida_core/.travis-data/workchains.py", line 77, in do_run
    self.out('output', echo_inline(input_string=self.inputs.input)[1]['output'])
  File "/home/greschd/programming/aiida/aiida_core/aiida/orm/calculation/inline.py", line 159, in swap_result
    result = wf.run_get_node(*args, **kwargs)
  File "/home/greschd/programming/aiida/aiida_core/aiida/work/workfunctions.py", line 63, in run_get_node
    return proc.execute(), proc.calc
  File "/home/greschd/programming/aiida/aiida_core/aiida/work/processes.py", line 618, in execute
    result = super(FunctionProcess, self).execute(return_on_idle)
  File "/home/greschd/.virtualenvs/aiida_dev/src/plumpy/plumpy/processes.py", line 406, in execute
    return Executor(return_on_idle).execute(self)
  File "/home/greschd/.virtualenvs/aiida_dev/src/plumpy/plumpy/processes.py", line 71, in execute
    return loop.run_sync(lambda: self._future)
  File "/home/greschd/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/tornado/ioloop.py", line 453, in run_sync
    self.start()
  File "/home/greschd/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/tornado/ioloop.py", line 755, in start
    raise RuntimeError("IOLoop is already running")
RuntimeError: IOLoop is already running
```

**Comment by greschd** [comment]
Somewhat surprisingly, the same problem doesn't appear to occur when running a ``workfunction`` instead of an ``InlineCalculation``. That's surprising because the ``InlineCalculation`` is wrapped in a ``workfunction`` for processing.

**Comment by sphuber** [comment]
Fixed in PR #1252
